projects/angular-split/src/lib/custom-gutter.ts

```diff
@@ -0,0 +1,195 @@
+import {
```

Comment:
It all looks solid, I think the only thing I'd like to see changed is to split this file up into separate files (basically 1 directive per file), they can maybe live in the `projects/angular-split/src/lib/gutter` directory.